pages/common/ajson.md

```diff
@@ -0,0 +1,24 @@
+# ajson
+
+> Executes JSONPath with JSON document.
+> More information: <https://github.com/spyzhov/ajson>.
+
+- Read JSON from a file and execute JSONPath:
+
+`ajson "$..json[?(@.path)]" example.json`
+
+- Read JSON from STDIN and execute JSONPath:
+
+`cat example.json | ajson "$..json[?(@.path)]"`
+
+- Read JSON from `GET` request and calculate value:
+
+`ajson "avg($..price)" "https://example.com/api/"`
+
+- Curl JSON from `POST` request and calculate value:
```

Comment:
This example seems to be demonstrating the same functionality of `ajson` as the previous stdin example. Generally in tldr pages we try to focus on the functionality of the command itself and not try to show examples of other commands. Perhaps this example could be removed?